.eslintrc

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
I believe it makes sense to base on companies linting rules https://github.com/hellofresh/eslint-config-senf